src/util/swipe.ts

```diff
@@ -0,0 +1,24 @@
+import "hammerjs";
+
+export const setupHorizontalSwipe = (backwardsCallback, forwardsCallback) => {
+  const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
+
+  if (!isMobile) return () => {};
+
+  const mc = new Hammer.Manager(document.body);
+  mc.add(
+    new Hammer.Swipe({
+      threshold: 120,
+      velocity: 1,
```

Comment:
These parameters are much larger than the defaults so I'm just concerned it requires a swipe to be faster and longer than is typical.  That said, I cannot find any documented guidelines.  [Apple's swipe recognizer](https://developer.apple.com/documentation/uikit/uiswipegesturerecognizer) describes fast and slow possibilities but has no useful details.

@matthiasdebaat do you know where there are any UX guidelines on the gritty specifics of gestures?